docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -5,6 +5,18 @@
 
 {description}
 
+== Best Practices
+
+Before starting a rolling upgrade, consider the following best practises:
+
+* Ensure that the <<hazelcast-members-compatibility-guarantees,Compatibility Guarantees>> cover your requirements
+* If the minimum version of JDK has changed between releases, <<updated-jdk,upgrade the JDK version>> before starting a rolling upgrade
+* If you used Hazelcast Operator to deploy your cluster, refer to the link:https://docs.hazelcast.com/operator/latest/scaling-upgrading#upgrading[Operator documentation, window=_blank]
```

Comment:
Trying to avoid a big naming thread but should 1st ref be Hazelcast Platform Operator? I might be out of date on this...

Reply:
The (last) rename that I saw dropped Platform to make it just Hazelcast Operator - and this also matches the [current] naming in the side nav of the docs.

Actually, I think the last one I saw was Hazelcast **Enterprise** Operator, but Platform was definitely not part of the name.

I suggest leaving as is - if it needs to be changed, we can do that anytime.